supervisely/app/widgets/custom_models_selector/custom_models_selector.py

```diff
@@ -385,19 +390,35 @@ def _generate_table_rows(self, checkpoint_infos: List[CheckpointInfo]) -> List[D
                     checkpoint=checkpoint_info,
                     task_type=checkpoint_info.task_type,
                 )
-                table_rows.append(model_row)
+                table_rows[checkpoint_info.task_type].append(model_row)
             except:
                 continue
+        table_rows = dict(table_rows)
         return table_rows
 
```

Comment:
It's not very convenient to read this sorting function.
As an example we can define the "default" order:

```python
def _filter_task_types(self, task_types: List[str]):
    order = ["object detection", "instance segmentation", "pose estimation"]
    sorted_tt = [task for task in order if task in task_types]
    other_tasks = sorted(set(task_types) - set(order))
    sorted_tt.extend(other_tasks)
    return sorted_tt
    ```